packages/containers/src/PieChartButton/__snapshots__/PieChartButton.test.js.snap

```diff
@@ -1,10 +1,27 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Filter container should render 1`] = `
+exports[`PieChartButton container should render 1`] = `
 <PieChartButton
   display="small"
   inProgress={undefined}
   labelIndex={0}
+  loading={undefined}
+  minimumPercentage={5}
+  model={undefined}
```

Comment:
Hmm it's weird, you should have your array of elements from initial state right ?